test/distributed/_tensor/test_random_ops.py

```diff
@@ -126,6 +175,87 @@ def test_deterministic_dropout_1d(self):
                         local_tensor[self_slice, :], local_tensor[other_slice, :]
                     )
 
+    @with_comms
+    @skip_unless_torch_gpu
+    def test_deterministic_uniform_nd(self):
+        mesh = torch.arange(self.world_size).reshape(2, 2, -1)
+        device_mesh = DeviceMesh(self.device_type, mesh)
+        _local_tensor = torch.empty(
+            *[self.world_size for _ in mesh.size()], device="cuda"
+        )
+        # initialize rng state
+        manual_seed(1234, device_mesh)
+        self.check_rng_state(1234, 0, device_mesh)
+
+        placements_list = [  # this list of placements should be enough to cover
+            [Shard(0), Shard(1), Shard(2)],
+            [Shard(2), Shard(1), Shard(0)],
+            [Shard(1), Replicate(), Shard(0)],
+            [Replicate(), Replicate(), Replicate()],
+        ]
+        dim_map_list = [
```

Comment:
I think we don't need the dim_map_list and test whether it would be equal to `spec.dim_map`? It's already been tested in all other tests.